ansible/roles/test/tasks/snmp/interfaces.yml

```diff
@@ -25,9 +34,14 @@
 
 - debug: var=minigraph_map_sonic_to_ngs
 - debug: var=snmp_intf
+- debug: var=snmp_mgmt_intf
 - debug: var=mg_intf
 
 - name: Check for missing interfaces in SNMP
   fail: msg="Minigraph interface {{ minigraph_map_sonic_to_ngs[item] if item in minigraph_map_sonic_to_ngs else item }} not in SNMP interfaces"
   when: "{{ (item in minigraph_map_sonic_to_ngs and minigraph_map_sonic_to_ngs[item] not in snmp_intf) or (item not in minigraph_map_sonic_to_ngs and item not in snmp_intf) }}"
   with_items: "{{ mg_intf }}"
+
+- name: Check for missing management interfaces in SNMP
```

Comment:
>Check for missing management interfaces in SNMP [](start = 8, length = 47)

From SNMP client point of view, front panel port and management port are the same. Can we check for missing frontpanel/management interfaces in SNMP in a single loop, without differentiate them first?